### ISSUE TITLE
Uncaught (in promise) TypeError: this.pendingRequest.dispose is not a function

### ISSUE THREAD

**Issue by ohardy** [4Catalyzer/found-relay#92]
With `0.3.0-alpha.8` version this bug just appear. If I click on a Link while a query is running, I got this error:
```
Uncaught (in promise) TypeError: this.pendingRequest.dispose is not a function
    at QuerySubscription.dispose (http://localhost:3000/static/js/bundle.js:14996:27)
    at http://localhost:3000/static/js/bundle.js:15411:27
    at Array.forEach (<anonymous>)
    at Resolver.updateQuerySubscriptions (http://localhost:3000/static/js/bundle.js:15409:33)
    at Resolver._callee$ (http://localhost:3000/static/js/bundle.js:15291:41)
    at tryCatch (http://localhost:3000/static/js/bundle.js:5403:40)
...
```

**Comment by taion** [comment]
oops

**Comment by taion** [comment]
Fix released in v0.3.0-alpha.9.
